Library/Formula/libsearpc.rb

```diff
@@ -0,0 +1,65 @@
+class Libsearpc < Formula
+  homepage "http://www.seafile.com/"
+  url "https://github.com/haiwen/libsearpc/archive/v3.0-latest.tar.gz"
+  sha256 "56313771e0ad7dc075c4590b6a75daeb3939937b21716d82c91be2612133b8cd"
+  version "3.0"
+
+  head "https://github.com/haiwen/libsearpc.git"
+
+  # FIX for homebrew autotools path
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!